nucliadb_models/src/nucliadb_models/labels.py

```diff
@@ -23,19 +23,20 @@
 
 from pydantic import BaseModel
 
-BASE_LABELS: dict[str, list[str]] = {
-    "t": [],  # doc tags
-    "l": [],  # doc labels
-    "n": [],  # type of element: i (Icon). s (Processing Status)
-    "e": [],  # entities e/type/entityid
-    "s": [],  # languages p (Principal) s (ALL)
-    "u": [],  # contributors s (Source) o (Origin)
-    "f": [],  # field keyword field (field/keyword)
-    "fg": [],  # field keyword (keywords) flat
-    "m": [],  # origin metadata in the form of (key/value). Max key/value size is 255
-    "p": [],  # origin metadata in the form of (key/value). Max key/value size is 255
-    "k": [],  # kind of text paragraph to be stored
-    "q": [],  # reserved for internal use: h (hidden)
+BASE_LABELS: dict[str, set[str]] = {
+    "t": set(),  # doc tags
```

Comment:
Just a curiosity, you can also use `{}` instead of `set()`.

Reply:
python interprets this `{}` as empty dict 

```python
>>> base = {"t": {}}
>>> type(base["t"] )
<class 'dict'>
```